### DISCUSSION TITLE
Rhino w/out Node.js

### DISCUSSION THREAD

**Discussion by bdav56** [Appsilon/rhino#585] [Q&A]
I followed along during a Rhino presentation and I thought the Rhino framework. The one exception is I do not want to install Node.js or code in any language other than R - my preference is to do adjust the formatting the standard Shiny way. Are there any issues with running Rhino applications without any Sass or JavaScript? Can I delete the formatting files that Rhino automatically creates?

**Comment by PawanRamaMali** [comment]
In short, Rhino allows you to create Shiny apps with a focus on software engineering best practices. While it encourages using Sass (for styling) and JavaScript (for interactivity), it doesn’t enforce them. You can definitely build Rhino applications without Sass or JavaScript. 

So, there are no inherent issues with running Rhino apps without Sass or JavaScript but consider the trade-offs in terms of aesthetics and functionality.

You can indeed delete the files that Rhino automatically creates. To switch from Node.js-based Sass to R-based Sass, modify the `rhino.yml` configuration file by changing `sass: node` to `sass: r`
